compiler-rt/cmake/Modules/AllSupportedArchDefs.cmake

```diff
@@ -20,6 +20,7 @@ set(VE ve)
 if(APPLE)
   set(ARM64 arm64)
   set(ARM32 armv7 armv7s armv7k)
+  set(ARM64_32 arm64_32)
```

Comment:
I am confused why this would be new here. We already have watch support for Asan. I understand the other changed line. But this is surprising we need this.